executor-service/src/test/java/executor/service/params/ThreadPoolConfigParams.java

```diff
@@ -0,0 +1,67 @@
+package executor.service.params;
+
+import executor.service.model.ThreadPoolConfig;
+import org.junit.jupiter.params.provider.Arguments;
+
+import java.util.stream.Stream;
+
+public class ThreadPoolConfigParams {
+
+    static Stream<Arguments> testAllArgsConstructor() {
+        return Stream.of(
+                // expectedCorePoolSize, expectedKeepAliveTime
```

Comment:
It might be better to use DisplayName instead of comments

Reply:
The hint is just for the readability of how the actual parameters look like in the test